ugandaemr_mobile/uninstalling-ugandaemr-mobile.md

```diff
@@ -0,0 +1,2 @@
+# Uninstalling UgandaEMR Mobile
```

Comment:
@solemabrothers This page is empty - delete it

Reply:
Done